acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/SignerConfigurationBuilder.java

```diff
@@ -33,6 +33,9 @@
   private List<String> httpHostAllowList = emptyList();
   private TlsOptions serverTlsOptions;
   private TlsCertificateDefinition overriddenCaTrustStore;
+  private String slashingProtectionDbUrl = "";
```

Comment:
do we have checks which ensure if a username is specified, then all other fields must be too?